### PR TITLE
Triangle spectrum and radon transform

### PR DESCRIPTION
**Pull request by JeffFessler**

### CODE REVIEW

File: src/triangle.jl [hunk 1]
```diff
@@ -3,57 +3,174 @@ triangle.jl
...
 =#
 
-using ImagePhantoms #: Object2d
+
+using ImagePhantoms #: Object, Object2d
 
 export Triangle
+export phantom, radon, spectrum
+
+const sqrt3 = sqrt(3)
```
Comment (johnnychen94) [inline src/triangle.jl:12]: If this is for performance consideration, then it is not necessary; the Julia compiler would just optimize these constant operations.

**Review by johnnychen94 — APPROVED** [review]
Nothing suspicious to me.

File: src/triangle.jl (hunk 1 above)
Comment (JeffFessler) [reply src/triangle.jl:12]: It's mostly to save typing :)

**Comment by JeffFessler** [comment]
Implementing the spectrum and Radon transform of triangles was more work than I had expected, but it is done now and maybe someone will find it useful eventually...

**Comment by johnnychen94** [comment]
Maybe you want to add a reference to this package in TestImages?

**Pull request merged**